### PR TITLE
Adds Reckon mob verb, Adds mindread back as different way

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
* Adds Reckon mob verb
* Adds mindread back as different way

### Reackon
Reckon works similarly to pray, but it's only visible to you and ghosts.  You can use it as an RP tool for expressing how you think for a situation, or used to say something to ghosts without braking weird IC consequences.
The difference between reckon and pray is that how you describe a situation. Pray is supposed to pray literally, while reckon is thking yourself, so you'd write a sentence in a very different manner. Admin can possibly help you if they read your reckon, but it will be less impact than pray.

### Mindread
Mindread is back, but it will automatically read nearby people's reckon. If you're hynpotise vulnerable while you hear someone's hypnotic thought, you'll be hypnotised, so don't be nosy.
I didn't add mindread-able feature to all telepathic methods because I am worried of some validhunting (i.e. cult telepathy, holopara telepathy, etc) We'll see if it's fine to be added

## Why It's Good For The Game
More RP tool

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: Reckon verb is newly added. It works similarly like how pray verb works, but the message is only visible to ghosts, but also mindread mutation holder.
add: Mindread mutation is back. You'll automatically read nearby people's reckon, but there is a chance that you can be hypnotised when you mindread someone's hypnotic message.
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
Possible ideas:
* some ways that can prevent your reckon to be mindread
* Foilhat prevents your reckon to be mindread
* If things above are validated, it's fine to add more mindreadable reckons

**Comment by Absolucy** [comment]
Heres some ideas I had:

- Perhaps the psychiatrist gimmick role should be mindread-resistant - reading their thoughts should be less common or perhaps garbled.
- Mindshield preventing mindread wouldn't make sense, tbh.
- Perhaps have different "levels" of mindread, and stick holopara chat and cult telepathy and such behind a currently admin-only level of mindread, just in case any admins get an idea for a gimmick.
  - Maybe have higher-level mindread capable of _rarely_ detecting brainwashes, perhaps in a garbled form?
- Mindreading someone with really low sanity should apply a negative moodlet from being 'exposed' to their lack of sanity.

**Comment by Tsar-Salat** [comment]
Perhaps theres a way to include Revenants/other spectator mobs?

It could help with our recent addition to make those roles less shitter and more rp

File: code/datums/mutations/mindread.dm [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/datum/mutation/mindread
+	name = "Mindread"
+	desc = "A mutation that allows the user to read neayby people's thoughts and prays."
```
Comment (Absolucy) [inline code/datums/mutations/mindread.dm]:
```suggestion
	desc = "A mutation that allows the user to read nearby people's thoughts and prays."
```

**Comment by PowerfulBacon** [comment]
The only purpose this seems to serve to me is so that you can advertise to ghosts when you are doing something interesting and want them to watch

**Comment by EvilDragonfiend** [comment]
> The only purpose this seems to serve to me is so that you can advertise to ghosts when you are doing something interesting and want them to watch

If they want to do, they can do, and it's better than saying to air even in RP-wise.
also, reckon has a limited way to let people communicate. For example, if you're deaf and say something, your message will be delivered to nearby mindreaders 'someone wanted to say bla bla.'

**Comment by PowerfulBacon** [comment]
I'm not sure how I feel about this, I don't think 2 different versions of the same verb will be used by people and I doubt it will be used for RP purposes given the fact that 95% of the time, nobody will ever see your reckoning messages.

**Comment by EvilDragonfiend** [comment]
> I'm not sure how I feel about this, I don't think 2 different versions of the same verb will be used by people and I doubt it will be used for RP purposes given the fact that 95% of the time, nobody will ever see your reckoning messages.

to be honest, it's a fair point. there should be more mechanics with this reckoning thing even if players are not going to use it actively.

**Pull request closed without merging**